be/src/vec/exprs/vexpr.cpp

```diff
@@ -653,7 +653,7 @@ Status VExpr::_evaluate_inverted_index(VExprContext* context, const FunctionBase
                         context->get_inverted_index_context()
                                 ->get_storage_name_and_type_by_column_id(column_id);
                 auto storage_type = remove_nullable(storage_name_type->second);
-                auto target_type = cast_expr->get_target_type();
+                auto target_type = remove_nullable(cast_expr->get_target_type());
```

Comment:
Why add remove_nullable?

Reply:
remove null to get real type, it's a bug in original code